examples/flava/data/datamodules.py

```diff
@@ -33,12 +33,20 @@
     VLTransform,
 )
 from .utils import build_datasets_from_info, fetch_images
+from torch.utils.data.distributed import DistributedSampler
+import torch.distributed as dist
 
 
 def transform_image(transform, sample):
     sample.update(transform(sample["image"]))
     return sample
 
+def get_sampler(dataset, shuffle=True):
+    if dist.is_initialized():
+        return DistributedSampler(dataset, shuffle=shuffle)
+    if shuffle:
+        return torch.utils.data.RandomSampler(dataset)
+    return torch.utils.data.SequentialSampler(dataset)
```

Comment:
this doesn't affect lightning training, but let me know if you want me to revert it @ankitade